README.md

```diff
@@ -133,6 +133,13 @@ Veneur uses [HyperLogLogs](https://github.com/axiomhq/hyperloglog) for approxima
 
 Via an optional [magic tag](#magic-tag) Veneur will forward counters to a global host for accumulation. This feature was primarily developed to control tag cardinality. Some counters are valuable but do not require per-host tagging.
 
+# Terminology
+
+* **Emitting:** Sending a metric from a non-Veneur instance to a Veneur instance. Example: emitting metrics from the veneur-emit CLI to a local Veneur instance
```

Comment:
As previously mentioned, I think these are reasonably clean delineations - but it would be good to get a second opinion from the team. Or perhaps @sjung-stripe might be able to weigh in here?

Reply:
I will remove these from this commit then and re-add them at a time when we have gathered a wider opinion.